chunky/src/java/se/llbit/util/ProgressListener.java

```diff
@@ -17,23 +17,25 @@
  */
 package se.llbit.util;
 
+import java.time.Duration;
+
 /**
  * Progress listener.
  *
  * @author Jesper Öqvist <jesper@llbit.se>
  */
 public interface ProgressListener {
-  ProgressListener NONE = (task, done, start, target) -> {};
+  ProgressListener NONE = (task, done, start, target, duration) -> {};
 
   /**
    * Update progress without ETA.
    */
-  void setProgress(String task, int done, int start, int target);
+  void setProgress(String task, int done, int start, int target, Duration duration);
 
   /**
    * Update progress with ETA.
    */
-  default void setProgress(String task, int done, int start, int target, String eta) {
-    setProgress(task, done, start, target);
+  default void setProgress(String task, int done, int start, int target, Duration duration, Duration eta) {
```

Comment:
Same here - is eta "Estimated Time of Arrival" the remaining duration or the total estimated duration of the task?
```suggestion
  default void setProgress(String task, int done, int start, int target, Duration duration, Duration remainingTime) {
```

Reply:
I even considered changing this to an `Instant` that has the actual _time_ due to the strict meaning of "ETA" but then decided that we never show the actual time and if we do, we can just calculate it from the remaining time. 

Renaming is a great idea, as above.